src/types.ts

```diff
@@ -253,3 +262,13 @@ export interface ProgramWorkflow {
     uuid: string;
   };
 }
+
+export interface DatePickerTypeOption {
+  value: DatePickerType;
+  label: string;
+  defaultChecked: boolean;
+}
+
+export interface DatePickerTypeOptions {
+  [key: string]: Array<DatePickerTypeOption>;
+}
```

Comment:
If we don't plan to define any props explicitly, then `Record<string, Array<DatePickerTypeOption>>;` would suffice